### DISCUSSION TITLE
IPYNB to PDF

### DISCUSSION THREAD

**Discussion by usamabajwa86** [AammarTufail/python_ka_chilla2023#63] [Polls]
Exporting IPYNB to PDF using VS Code

**Comment by Saibtain** [comment]
i think phely to easily pdf bn jati thio ab nhi b n rahi . 
yee software bhi istall kia ha but no response  koi bahi just screen recording kr ka upload kr sakta ha k pdf kesa bnata han

**Reply by muhammadadeelsaif** [reply]
kya ap detail se bata sakte hen ke ipynb se pdf kese convert krni he yaha opar to koi treeqa nahi mila muje.

**Reply by usamabajwa86** [reply]
Follow these steps

https://github.com/AammarTufail/python_ka_chilla2023/discussions/29
